### PR TITLE
Update and upgrade messages

### LINKED ISSUE #10146: Add the alert string for 0.16 users to switch from IE11 to a more recent browser
Reminder not to forget this before the string freeze this week!
cc @marcellamaki 

## Observed behavior
…

## Errors and logs
…

## Expected behavior
…

## User-facing consequences
…

## Steps to reproduce
…

## Context
…

### PR DESCRIPTION
**Pull request by marcellamaki**
for IE 11, python 2.7 and the telemetry notification (intentionally somewhat vague)

### CODE REVIEW

File: kolibri/core/assets/src/mixins/commonCoreStrings.js [hunk 1]
```diff
@@ -1088,6 +1088,30 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
...
   },
 
+  // Device upgrades recommended
+  currentDeviceUsingIE11: {
+    message: 'You seem to be using Internet Explorer 11.',
+    context:
+      'Displayed on a device that is using Internet Explorer 11, as part of a message encouraging the user to upgrade.',
+  },
+  userDevicesUsingIE11: {
+    message: 'Users accessing Kolibri seem to be using Internet Explorer 11',
```
Comment (rtibbles) [inline kolibri/core/assets/src/mixins/commonCoreStrings.js]: Should be consistent here and have a `.` at the end to match the string above.

**Review by rtibbles — DISMISSED** [review]

File: kolibri/core/assets/src/mixins/commonCoreStrings.js [hunk 2]
```diff
@@ -1088,6 +1088,30 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
...
+    message:
+      'Please note that  support for this browser will be dropped in the upcoming version 0.17, so please install another browser to continue working with Kolibri. We recommend Mozilla Firefox or Google Chrome.',
+    context:
+      'Displayed to users of kolibri where one or more devices on the network are using Internet Explorer 11, as part of a message encouraging the user to upgrade.',
+  },
+  pythonSupportWillBeDropped: {
+    message:
+      'Please note that support for Python 2.7 will be dropped in the upcoming version 0.17. Upgrade your Python version to Python 3.7+ to continue working with Kolibri. More recent versions of Python 3 are recommended.',
+    context:
+      'Displayed to users of kolibri where one or more devices on the network are using Internet Explorer 11, as part of a message encouraging the user to upgrade.',
```
Comment (radinamatic) [inline kolibri/core/assets/src/mixins/commonCoreStrings.js]: This string is about Python, but the context still refers to IE11?

Comment (rtibbles) [reply kolibri/core/assets/src/mixins/commonCoreStrings.js]: Good catch Radina!

**Review by radinamatic — APPROVED** [review]
All the most recent recommendations are in, good to go! 👍🏽 💯 :shipit: 
Thank you!

**Pull request merged**